maps/ML-TTRevathi.map

```diff
@@ -149,15 +149,15 @@ u=ൗ
 ß=ത്മ    
 à=ക്ത    
 á=ഗ്ന    
-â=ന്‍റ    
+â=ൻറ    
```

Comment:
Is this fine? Should it be ൻ്റ or ന്റ?

Reply:
ya. it should be ന്റ. whoever created this map wrongly type ന്റ. will fix that too

Reply:
is there any other occurance of it?